change/@fluentui-utilities-2021-01-20-13-19-09-chore-bumping-versions.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "prerelease",
+  "comment": " Updating dependencies (react typings and formik.)",
```

Comment:
Ideally formik shouldn't be mentioned in these change files, since it's only used by northstar

Reply:
And probably the northstar changelog should mention the formik update/which packages it applies to

Reply:
i've updated the language.